src/main/scala/com/actian/spark_vector/colbuffer/decimal/DecimalIntColumnBuffer.scala

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I would keep them in separate files because of the relationship to the object companions (easier to see the file <-> ColumnBuffer's list relation when looking at the project's file hierarchy)